qa/vagrant-helpers.rb

```diff
@@ -1,15 +1,26 @@
 # encoding: utf-8
 require "open3"
+require "json"
 
 module LogStash
   class VagrantHelpers
+    DEFAULT_CENTOS_BOXES = [ "centos-6", "centos-7" ]
+    DEFAULT_DEBIAN_BOXES = [ "ubuntu-1204", "ubuntu-1404"]
 
-    def self.bootstrap
-      execute("vagrant up")
+    def self.bootstrap(box)
+      Dir.chdir("acceptance") do
+        p execute("vagrant up #{box} --provider virtualbox")
+        raw_ssh_config    = execute("vagrant ssh-config")[:stdout].split("\n");
+        parsed_ssh_config = parse(raw_ssh_config)
+        File.write(".vm_ssh_config", parsed_ssh_config.to_json)
```

Comment:
Just a note, using static port and ips is a old strategy when I was doing ansible works.